templates/ces_admin/ces-dashboard.html

```diff
@@ -5,36 +5,45 @@
 
 <div class='row'>
     <div class='col-md-10 offset-md-1'>
-        <div class='row mt-4'>
+        <div class='row mt-6'>
             <div class='col-md-12'>
-                <h2>Open sessions</h2>
-                <h4><span class='badge badge-pill badge-primary'>{{ open_sessions }}</span> users have open sessions</h4>
-                <p>These users started a survey in the last 24 hours.</p>
-                {% if open_sessions %}
-                    <div id="open-chart" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
-                {% endif %}
+                <h2>Overview</h2>
+
+                <div class='row'>
+                    <div class='col-md-4 p-5 text-center'>
+                        <h3>{{ open_sessions }}</h3>
+                        <p>Surveys currently in progress (started in the last 24 hours)</p>
+                    </div>
+                    <div class='col-md-4 p-5 text-center'>
+                        <h3>{{ percentage_complete | floatformat }}%</h3>
+                        <p>Percentage of surveys started that were completed</p>
+                    </div>
+                    <div class='col-md-4 p-5 text-center'>
+                        <h3>{{ percentage_canceled | floatformat }}%</h3>
+                        <p>Percentage of surveys started that were canceled</p>
+                    </div>
+                </div>
             </div>
         </div>
-        <div class='row mt-4'>
+        <div class='row mt-6'>
             <div class='col-md-12'>
-                <h2>Canceled sessions</h2>
-                <h4><span class='badge badge-pill badge-primary'>{{ canceled_sessions }}</span> users canceled their session</h4>
-                <p>These users canceled their session, either by instructing the tool to close thier session (e.g. by typing "end") or by abandoning their session after 24 hours. This chart shows the question or resource on which users canceled their sessions.</p>
-                <div id="canceled-chart" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
+                <h2>How did users end their sessions?</h2>
+                <h4>Completed sessions <span class='badge badge-pill badge-primary'>{{ completed_sessions }}</span></h4>
+                <p>These users completed the entire survey! This chart shows the last message or resource sent to the user.</p>
```

Comment:
Is it possible to cut these tooltips to just the beginning, i.e. not displaying all the specific phones numbers? It feels distracting and makes them hard to distinguish from each other.

Reply:
I am going to ask Stephanie her preference. I see the value in having the ENTIRE message text, which could help swiftly analyze the data, without referring to the data spreadsheet. But I agree that it's a lot to take in.